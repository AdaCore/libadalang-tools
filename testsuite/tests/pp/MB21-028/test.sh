gnatpp --pipe p.adb
gnatpp --pipe p2.adb
gnatpp --pipe q.adb
