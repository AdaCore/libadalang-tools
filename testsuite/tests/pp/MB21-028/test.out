
WARNING: GNATpp will be baselined soon. Please switch to GNATformat, the new Ada code formatter, for future use.
-------

procedure P is
   pragma Warnings (Off);
   --  Explain why we turned warnings off.
begin
   null;
end P;

WARNING: GNATpp will be baselined soon. Please switch to GNATformat, the new Ada code formatter, for future use.
-------

procedure P is
   pragma Warnings (Off);

   --  Explain why we turned warnings off.

begin
   null;
end P;

WARNING: GNATpp will be baselined soon. Please switch to GNATformat, the new Ada code formatter, for future use.
-------

package body Q is

   procedure P is
      pragma Warnings (Off);
      --  Explain why we turned warnings off.
   begin
      null;
   end P;

   procedure P is
      pragma Warnings (Off);
      --  Explain why we turned warnings off.

   begin
      null;
   end P;

   procedure P is
      pragma Warnings (Off);

      --  Start of processing for P.
   begin
      null;
   end P;

   procedure P is
      pragma Warnings (Off);

      --  Start of processing for P.

   begin
      null;
   end P;

end Q;
